### PR TITLE
[OY2-19479] Create USA Banner Component

### PR DESCRIPTION
**Pull request by jphechter**
## Purpose

Create USA Banner Component

## Type of Request

feature

## Approach

Implements the standard USA Banner seen in both CMS Component Library and USWDS.

## Assorted Notes/Considerations

This component also relies on the `uswds.js` script for show/hide functionality. Due to issues working with this file in Storybook and Jest the component may function inconsistently in these environments. When we are able to address the problems in these places, we will need to update testing for this component.

#### Pull Request Creator Checklist

- [X] It is documented what type of request this is.
- [X] There is a detailed description for the purpose of this PR.
- [X] This PR is in line with the intention of this Project

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -3,12 +3,38 @@ import "./assets/js/uswds-init.min.js";
...
 import { Button } from "./components/Button/Button";
 import { Card } from "./components/Card/Card";
+import { Checkbox } from "./components/Checkbox/Checkbox";
+import { Dropdown } from "./components/Dropdown/Dropdown";
+import { Footer } from "./components/Footer/Footer";
+import { Header } from "./components/Header/Header";
 import { Icon } from "./components/Icon/Icon";
-import { Logo } from "components/Logo/Logo";
-import { TextArea } from "components/TextArea/TextArea";
-import { TextInput } from "components/TextInput/TextInput";
```
Comment (jphechter) [inline src/index.tsx:11]: The absolute imports in this file break the components when using the NPM package. Added a leading `./` where needed.

File: src/index.tsx [hunk 2]
```diff
@@ -3,12 +3,38 @@ import "./assets/js/uswds-init.min.js";
...
+  Header,
+  Icon,
+  Link,
+  Logo,
+  Radio,
+  RadioGroup,
+  TextArea,
+  TextInput,
+  Typography,
+  UsaBanner,
```
Comment (jphechter) [inline src/index.tsx:39]: Added missing components.

File: src/utils.tsx [hunk 3]
```diff
@@ -1,3 +1,33 @@
...
+      script.src = "/assets/js/uswds.js";
+      document.body.appendChild(script);
+      return () => {
+        // clean up effects of script here
+      };
+    }, []);
+
+    return isLoaded ? <Story /> : <div>Loading...</div>;
+  },
+];
```
Comment (jphechter) [inline src/utils.tsx:33]: Pulled from #82

**Review by jdinh8124 — APPROVED** [review]
Looks good to me make sure to pop off the scss file from the PR

**Review by jdinh8124 — APPROVED** [review]
Looks good to me

**Pull request merged**